src/components/projects/projects-table/projects-table/projects-table.tsx

```diff
@@ -24,21 +25,89 @@ interface DisplayProjectProps {
   allProjects: DisplayProject[];
 }
 
+/***
+ * Custom sorting order for wbsNums according to car, then project, then workPackage.
+ * @param a 1st wbsNum in string form
+ * @param b 2nd wbsNum in string form
+ * @param order Imported SortOrder values 'asc' or 'desc'
+ * @return number A number -1, 0, or 1 describing the value of a relative to b,
+ *                according to the specified SortOrder.
+ */
+function wbsNumSort(a: string, b: string, order: SortOrder): number {
```

Comment:
You might be able to take advantage of type inference and not need to explicitly define the output type.